setuptools/command/easy_install.py

```diff
@@ -491,7 +491,7 @@ def check_site_dir(self):  # noqa: C901  # is too complex (12)  # FIXME
             try:
                 if test_exists:
                     os.unlink(testfile)
-                open(testfile, 'w').close()
+                open(testfile, 'wb').close()
```

Comment:
"Touch" operation... can be done in the binary mode to avoid messing with encoding.